package/metrics_daemon/metrics_daemon/src/metrics_daemon.h

```diff
@@ -0,0 +1,16 @@
+//
+// Created by Yi Yang on 9/7/18.
+//
+
+#ifndef PIKSI_BUILDROOT_METRICS_DAEMON_H
+#define PIKSI_BUILDROOT_METRICS_DAEMON_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+json_object *jobj_root = NULL;
```

Comment:
In general, definitions that allocate storage should not be in header files since it places the symbol in whatever compilation unit includes it first.  Also, since this is not part of the external interface of the metrics daemon and it is externed in the unit tests, it should be moved into `metrics_daemon.c`.